### ISSUE TITLE
Accessibility [NVDA] A11y_MicrosoftGuides_EditStepCardText_ScreenReader: NVDA  does not announce the heading  "Scan Marker".

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10049]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:

Application Name: Microsoft Guides - PC Authoring 
Windows Version: Windows10

Desktop application version: 103.1906.18005.0

 

Steps to Reproduce:

1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using correct credentials.​
3. Select the instance and click on Continue.​
4. Navigate to 'Create a new Guide' button, add details and click on 'create' button.​
5. Navigate to 'Set your Anchor now' button and click on it.​
6. Navigate to 'Select' button under 'Printed Anchor' section and click on it.​
7. Navigate to 'Edit' tab Item.​
8. Navigate to 'Edit Step Card Text' button and click on it. ​
9.Start screen reader as NVDA

Actual Result: 
Nvda does not announce the heading for the pop-up "Scan Marker".
​
Expected Result: 
Nvda should read the heading when navigated to the heading "Scan Marker".

Recommendations:
Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

User Impact:
Screen reader user will not receive the information present on the screen.

MAS Reference:

MAS1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by DrSooom** [comment]
@fsteffi: Could you launch NVDA before starting Microsoft Guides - PC Authoring, because NVDA send a screenreader flag to the application. Maybe it could fix some issues if NVDA is running before starting an application.

**Comment by fsteffi** [comment]
@DrSooom  I have done exactly as you have asked , but the bug still reproduces with  NVDA 2019.2 RC2.

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
